### PR TITLE
UI: Added localization ui component part 1

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes :

- Added localization
- deleted unusead component

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.component.tsx [hunk 1]
```diff
@@ -248,7 +244,9 @@ const AddGlossary = ({
 
           <div>
             <div className="tw-flex tw-items-center tw-mt-4">
```
Comment (aniketkatkar97) [inline openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.component.tsx:246]: Tailwind stylings

File: openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.interface.ts [hunk 2]
```diff
@@ -13,3 +13,9 @@ export interface AddGlossaryProps {
   onSave: (data: CreateGlossary) => void;
   fetchTags?: () => void;
 }
+
+export enum AddGlossaryError {
+  NAME_REQUIRED = 'name required',
+  NAME_INVALID = 'name invalid',
+  DESCRIPTION_REQUIRED = 'description required',
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.interface.ts:20]: Localisation here too

File: openmetadata-ui/src/main/resources/ui/src/components/AddService/AddService.component.tsx [hunk 3]
```diff
@@ -17,8 +17,9 @@ import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { getServiceDetailsPath } from '../../constants/constants';
 import { GlobalSettingsMenuCategory } from '../../constants/GlobalSettings.constants';
+import { STEPS_FOR_ADD_SERVICE } from '../../constants/Ingestions.constant';
 import { delimiterRegex, nameWithSpace } from '../../constants/regex.constants';
-import { STEPS_FOR_ADD_SERVICE } from '../../constants/Services.constant';
+
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/AddService/AddService.component.tsx:22]: remove this empty line

**Review by aniketkatkar97 — COMMENTED** [review]
Left some comments. Mostly changes related to removing tailwind styling and localization. Can you have a look at that?

File: openmetadata-ui/src/main/resources/ui/src/utils/GlossaryUtils.ts [hunk 4]
```diff
@@ -402,3 +404,31 @@ export const getEntityReferenceFromGlossary = (
...
+      return errorMsg(
+        t('label.field-required', {
+          field: t('label.description'),
+        }) + '.'
+      );
+
+    default:
+      return;
+  }
+};
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/utils/GlossaryUtils.ts:434]:
```suggestion
export const getAddGlossaryError = {
    [AddGlossaryError.NAME_REQUIRED]: errorMsg(
        t('label.field-required', {
          field: `${t('label.glossary')} ${t('label.name-lowercase')}`,
        }) + '.'
      ),
    [AddGlossaryError.NAME_INVALID]: errorMsg(
        t('label.field-invalid', {
          field: `${t('label.glossary')} ${t('label.name-lowercase')}`,
        }) + '.'
      ),
     [AddGlossaryError.DESCRIPTION_REQUIRED]: errorMsg(
        t('label.field-required', {
          field: t('label.description'),
        }) + '.'
      )
  }
```

**Review by chirag-madlani — CHANGES_REQUESTED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.interface.ts (hunk 2 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/AddGlossary/AddGlossary.interface.ts:20]: this does not require localization as this is the enum keys, which we are using to get localized message from switch case!

**Comment by ShaileshParmar11** [comment]
> Left some comments. Mostly changes related to removing tailwind styling and localization. Can you have a look at that?

Hi @aniketkatkar97, removing tailwind does not come under the scope of this PR, as this is related to localization, we have a separate issue to handle the tailwind thing, let me know if you need any discussion around the same!

File: openmetadata-ui/src/main/resources/ui/src/components/AddService/AddService.component.tsx (hunk 3 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/AddService/AddService.component.tsx]: done

**Review by aniketkatkar97 — APPROVED** [review]

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request closed without merging**